pubspec.yaml

```diff
@@ -1,27 +1,28 @@
 name: app_versioning
 description: App versioning package
-version: 1.0.4
+version: 1.0.5
 publish_to: none
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.12.0 <4.0.0"
+  flutter: ">=3.10.0" # upgrader package requires this flutter min version
```

Comment:
Since we are forced to use flutter `3.10.0` and higher we can also enforce the dark sdk v3 and higher:
`sdk: ">=3.0.0 <4.0.0"`